riotfile.py

```diff
@@ -127,6 +127,22 @@ def select_pys(min_version: str = MIN_PYTHON_VERSION, max_version: str = MAX_PYT
                 "DD_CIVISIBILITY_ITR_ENABLED": "0",
             },
         ),
+        Venv(
+            name="appsec_iast_packages",
+            pys=["3.9", "3.10", "3.11", "3.12"],
```

Comment:
ooc, is there a reason for not testing with 3.8 and 3.13? for example, iast is not supported on those versions?

Reply:
Not that I know of, I just migrated what was already there, I can guess is to reduce CI usage? But now that it takes less time  (and since it runs in paralell) I will add 3.8.